HANDOFF.md

```
@@ -1214,3 +1214,11 @@ it), not a re-derivation of (a).
    - `hddGetPartitionInfo`: Added `nsub > APA_MAXSUB` clamping before copying sub-partitions to `parts[APA_MAXSUB + 1]`, preventing stack buffer corruption on invalid or corrupted APA partition headers.
    - `hddGetHDLGamelist`: Restored `saw_hdl` check to return `-ENOMEM` when partitions exist on disk but all memory allocations fail.
 
+# 28. STEP-202: Include regular ELFs in all 4 flavours in MEGA uploads (2026-08-15)
+
+### What changed:
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)